src/ontology/mondo-edit.obo

```diff
@@ -76573,10 +76573,10 @@ xref: UMLS:C0022680 {source="DOID:898", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C0085413 {source="NCIT:C84578", source="MONDO:notFoundInDiseaseSubset", xref="MONDO:equivalentTo"}
 xref: UMLS:CN119611 {source="MONDO:equivalentObsolete"}
 is_a: MONDO:0000426 {source="DOID:898", source="MONDO:Redundant"} ! autosomal dominant disease
-is_a: MONDO:0005240 {source="DOID:898", source="EFO:1001496/inferred", source="NCIT:C84578/inferred"} ! kidney disorder
 is_a: MONDO:0005372 ! male infertility
 is_a: MONDO:0019741 {source="EFO:1001496", source="MONDO:Redundant", source="Orphanet:730"} ! familial cystic renal disease
 is_a: MONDO:0020642 {source="NCIT:C84578"} ! polycystic kidney disease
+is_a: MONDO:0100313 {source="DOID:898", source="EFO:1001496/inferred", source="NCIT:C84578/inferred"} ! nephrogenetic disease
```

Comment:
@nicolevasilevsky risky this, as the isa changed, but the source annotations moved along with the change..